recipes/wally/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.2.3" %}
+{% set revision = "93442348984044ea12608c323b7480487ab70eba" %}
+
+package:
+  name: wally
+  version: '{{ version }}'
+
+source:
+  git_url: https://github.com/tobiasrausch/wally.git
```

Comment:
Does this have submodules? Otherwise it's better to use the tarball.

Reply:
Yes, I need the submodule for OpenCV.

Reply:
Let me try linking against conda opencv maybe that works then we can skip the submodules and use the release.

Reply:
The conda opencv didn't work for me. I stay with the submodules.